platform/device/internal/deviceproto/device.pb.go

```diff
@@ -1,21 +1,15 @@
-// Code generated by protoc-gen-go.
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
Here, too, I think this needs to be generated with `protoc-gen-go` instead of `-gogo`. I'm surprised you got this if you just ran `go generate`.